components/admin_console/team_channel_settings/team/list/team_list.scss

```diff
@@ -0,0 +1,20 @@
+.TeamsList {
+    .DataGrid {
+        .DataGrid_cell {
+    
+            .TeamList_NameText {
+                display: flex;
+                align-items: center;
+                height: 80px;
+            }
+        
+            .TeamList_ManagementText {
+                opacity: 0.65;
+            }
+        
+            .TeamList_EditText {
+                padding-right: 20px;
+            }
```

Comment:
Ditto with some of these names.

Reply:
And here:

```css
.TeamList_EditText {
```

should be

```css
.TeamList_editText {
```

Reply:
Done + Edited all other classnames to follow this pattern.